components/bookshelf/discover/SearchBox.tsx

```diff
@@ -33,17 +30,23 @@ const SearchBox = ({ setIsSearching }: SearchBoxProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  const handleChange = (e) => {
+  const handleChange = (e: {
+    target: { value: React.SetStateAction<string> }
+  }) => {
     setSearchInput(e.target.value)
   }
 
-  const handleSubmit = (e) => {
+  const handleSubmit = (e: { preventDefault: () => void }) => {
     e.preventDefault()
```

Comment:
Do you still need this?

Reply:
the type? or the function

Reply:
` e.preventDefault()`

Reply:
yeah, I just tested it without, and its refreshing the page. should I just use an `onClick` and move the function to a button? that way I can get rid of the prevent-default thing

Reply:
Oh ok. Hmmm Retain it for now